pkg/controller/namespacelabel/namespacelabel_controller.go

```diff
@@ -130,6 +143,15 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Watch the SKU rate limits config map
```

Comment:
Why is this watch on the SKU configmap added?

Reply:
That was a copy paste mistake on the comment. I've update it now.